services/web/server/src/simcore_service_webserver/diagnostics_monitoring.py

```diff
@@ -43,7 +43,9 @@ def middleware_factory(app_name: str) -> Coroutine:
             resp = await handler(request)
             log_exception = None
 
-            assert isinstance(resp, web.Response), "Forgot envelope middleware?"  # nsec
+            assert isinstance(
```

Comment:
MINOR: bandit is going to complain here. try to keep in a single line, e.g. by disabling black in the block. 
e.g. 
#fmt:off
assert isinstance(resp, web.Response), "Forgot envelope middleware?"  # nsec
#fmt:on